runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/utils/SparkTestPipelineOptionsForStreaming.java

```diff
@@ -28,12 +28,6 @@
  */
 public class SparkTestPipelineOptionsForStreaming extends SparkTestPipelineOptions {
 
-  @Override
```

Comment:
all of they used the SparkTestPipelineOptionsForStreaming so it can be removed, I hesitated about removing the whole class, because it only calls the method of the father, what do you think ?